src/book/01-team/01-introduction/01-introduction-to-nubots.mdx

```diff
@@ -18,13 +18,13 @@ The team is led and supported by academics and student team leaders.
 
 - **Associate Professor Stephan Chalup** (Computer Science and Engineering) - Head of the Newcastle Robotics Lab and NUbots
 - **Dr. Alexandre Mendes** (Computer Science and Software Engineering) - Deputy Head of NUbots
-- **Ysobel Sims** (Mathematics and Computer Science) - Student Co-Team Leader
-- **Josephus Paye II** (Computer Science) - Student Co-Team Leader
+- **Ysobel Sims** (Mathematics and Computer Science) - Student Team Leader
+- **Josephus Paye II** (Computer Science) - Mentor, Ex-Team Leader
 - **Alex Biddulph** (Computer Engineering) - Mentor, Ex-Team Leader
 - **Dr. Trent Houliston** (Software Engineering) - Mentor, Ex-Team Leader
 
 See [Current Team](/team/current-members) for our current team members and [Past Members](/team/past-members) for past members.
 
 ## The NUbots Lab
 
-The NUbots team works in the Robots and Brain Architecting Lab in ES115A. Feel free to stop by at any time for a tour!
+The NUbots team works in the Robots and Brain Architecture Lab in ES115A. Feel free to stop by at any time for a tour!
```

Comment:
Have you read the sign on the lab door?

Reply:
From memory, it should something be "Architecturing"

Reply:
I wasn't sure if it was a typo! I didn't even know Archtitecting was a word 😬

Reply:
I'd have to read the sign again, but in these times I cannot.

Reply:
I'll double check before finalising

Reply:
Architecting was correct.

Reply:
"Robotics & Brain Architecting Laboratory"

Reply:
I'd suggest changing it from 'Robot' to 'Robotics', and maybe lab to 'Laboratory' too, in line with what Kip said ^